CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 1.7.0 -- unreleased
+
+### Deprecation Warnings
+* Deprecated `MasterKey` and `MasterKeyProvider`. Replace your usage of these classes with `Keyring`. See `StandardKeyrings` 
+for the built-in keyrings that replace `KmsMasterKeyProvider`, `JceMasterKey`, and `MultiProviderFactory`. 
+We still support using master key providers and are not removing them yet. 
+When we decide to remove them, we will communicate that as defined in our versioning policy.
+* Deprecated `encryptData`, `decryptData` and related methods in `AwsCrypto`. Replace your calls to these methods with
+calls to `AwsCrypto.encrypt(EncryptRequest)` and `AwsCrypto.decrypt(DecryptRequest)`.
+
+### Minor Changes
```

Comment:
Yeah it sounded odd calling it minor, but I went with that since its a new minor version. "Features" doesn't show up in this changelog anywhere else. Where do we call keyrings "major"?

Reply:
I called it "major" in the Python changelog, but in retrospect, I'm not sure that's right either.

Looking back over our changelog history, things we have called "major" include:
* Introducing CMM concept
* Adding data key caching
* Deterministic IVs

Introducing keyring concept would fit into that general significance.

Reply:
I'm ok with major as long as we're ok with major feature != major version, which thinking about it seems fine since the versioning is more about backward compatibility.

Reply:
Yeah, our current versioning scheme slots major changes into the `Y` version increment, and reserves the `X` version increment for fundamental changes to the product.

Reply:
I've changed it to major